apps/mark-scan/frontend/src/pages/paper_reloaded_page.tsx

```diff
@@ -0,0 +1,27 @@
+import { Main, Screen, Text, H1 } from '@votingworks/ui';
+import { useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
+
+export function PaperReloadedPage(): JSX.Element {
+  // Messy, but this flow eventually ends up in BallotContext. By setting path now
+  // we ensure we render the correct page once poll worker auth is ended. If we don't
+  // set the path now, we'll incorrectly render PrintPage (the last ballot screen rendered).
+  const history = useHistory();
+  useEffect(() => {
+    history.push('/ready-to-review');
+  });
```

Comment:
I agree - if we can get rid of frontend routing it would be a big win. Not sure yet if doing that is blocked by migrating the remaining legacy frontend state to the backend. I think this change would be too big to fit into this PR so I've created https://github.com/votingworks/vxsuite/issues/3985 to track.

Reply:
Cool, I agree it's probably better to do as a separate refactor.